woodwork/statistics_utils/frequency_inference/_determine_most_likely_freq.py

```diff
@@ -0,0 +1,20 @@
+from ._constants import FREQ_INFERENCE_THRESHOLD, NON_INFERABLE_FREQ
+
+
+def _determine_most_likely_freq(alias_dict, threshold=FREQ_INFERENCE_THRESHOLD):
```

Comment:
The dosctrings for the other helper utils are very useful for understanding what's going on. Could you include one here as well?